wal/wal.go

```diff
@@ -223,6 +230,30 @@ func Create(lg *zap.Logger, dirpath string, metadata []byte) (*WAL, error) {
 	return w, nil
 }
 
+func (w *WAL) cleanupWAL(lg *zap.Logger) {
+	var err error
+	if err = w.Close(); err != nil {
+		if lg != nil {
+			lg.Panic("failed to closeup WAL during cleanup", zap.Error(err))
```

Comment:
Thanks for addressing my comments! Not trying to be picky but when you update the changes, I would suggest to use 'close WAL' instead of 'closeup WAL` in the log messages (here and on LOC #239)